packages/aws-cdk/README.md

```diff
@@ -372,6 +372,44 @@ For this reason, only use it for development purposes.
 **⚠ Note #2**: This command is considered experimental,
 and might have breaking changes in the future.
 
+### `cdk watch`
+
+The `watch` command is similar to `deploy`,
+but instead of being a one-shot operation,
+the command continuously monitors the files of the project,
+and triggers a `cdk deploy` whenever it detects any changes:
+
+```console
+$ cdk watch ExampleStack
```

Comment:
What happens if you don't specify a stack name? Preferably, the first example should be the minimal default usage:

```suggestion
$ cdk watch
```

Reply:
The problem is that no argument means "deploy everything", which doesn't really make sense for "watch".

I've changed the example to `cdk watch DevelopmentStack`, let me know if that's better.

Reply:
Why doesn't it make sense for watch to deploy everything?

Would that work?

Reply:
It does work, but `watch` is a development command - that's why it defaults `--hotswap` to `true`. For that reason, deploying _all_ stacks using it will be a rarity. In fact, for that reason, there might be a legitimate case for _not_ allowing all stacks to be deployed using `watch`, but not sure we want to go that far (it is useful for demos 😉).

Reply:
I think we should still support deploying all stacks with watch. There could be a case for dev apps that consist of multiple stacks